CODEOWNERS.txt

```diff
@@ -0,0 +1,42 @@
+# Add your github username to opt-in to automatic review requests for
+# specific sub-packages here:
+
+# astropy/constants
+# astropy/convolution
+# astropy/coordinates
+# astropy/io/ascii
+# astropy/io/fits
+astropy/io/misc             @WilliamJamieson
+# astropy/io/votable
+astropy/modeling            @WilliamJamieson
+# astropy/nddata
+# astropy/samp
+# astropy/stats
+# astropy/table
+# astropy/time
+# astropy/timeseries
+# astropy/uncertainty
+# astropy/units
+# astropy/utils
+# astropy/visualization
+# astropy/wcs
```

Comment:
Can we add an astropy/wcs/wcsapi entry here so that I can get assigned for reviews just for ``wcsapi`` but not all ``astropy/wcs``?